easybuild/tools/utilities.py

```diff
@@ -99,5 +102,9 @@ def import_available_modules(namespace):
                 mod_name = module.split(os.path.sep)[-1].split('.')[0]
                 modpath = '.'.join([namespace, mod_name])
                 _log.debug("importing module %s" % modpath)
-                modules.append(__import__(modpath, globals(), locals(), ['']))
+                try:
+                    mod = __import__(modpath, globals(), locals(), [''])
+                except ImportError as err:
```

Comment:
Ok if they're not for general use, but you might want to write more general functions and include them in vsc-base, so everybody benefits ;-)